db/database_test.go

```diff
@@ -671,17 +673,27 @@ func TestUpdateDesignDoc(t *testing.T) {
 	db := setupTestDB(t)
 	defer tearDownTestDB(t, db)
 
-	err := db.PutDesignDoc("official", DesignDoc{})
+	mapFunction := `function (doc, meta) { emit(); }`
+	err := db.PutDesignDoc("official", sgbucket.DesignDoc{
+		Views: sgbucket.ViewMap{
+			"TestView": sgbucket.ViewDef{Map: mapFunction},
+		},
+	})
```

Comment:
It still works -- I just retested it.  I added this due to the extra assertions added later in the test.